bittensor/metagraph.py

```diff
@@ -474,17 +473,15 @@ def _create_tensor(self, data, dtype) -> torch.nn.Parameter:
         # TODO: Check and test the creation of tensor
         return torch.nn.Parameter(torch.tensor(data, dtype=dtype), requires_grad=False)
 
-    def _set_weights_and_bonds(
-        self, root: bool = False, subtensor: bittensor.subtensor = None
-    ):
+    def _set_weights_and_bonds(self, subtensor: bittensor.subtensor = None):
         """
         Computes and sets weights and bonds for each neuron.
 
         Returns:
             None.
         """
         # TODO: Check and test the computation of weights and bonds
-        if root:
+        if self.netuid == 0:  # Is this the root network?
```

Comment:
Should this question be here in the code?

Reply:
Prob not. :D